src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/UserInviteRepositoryTests.cs

```diff
@@ -0,0 +1,116 @@
+﻿#nullable enable
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Areas.Identity.Data.Models;
+using GovUk.Education.ExploreEducationStatistics.Admin.Models;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using Xunit;
+using static GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services.DbUtils;
+
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services
+{
+    public class UserInviteRepositoryTests
+    {
+        private static readonly Guid _createdById = Guid.NewGuid();
+
+        [Fact]
+        public async Task CreateIfNoOtherUserInvite_role()
```

Comment:
Nitpick, but the casing for these test names is inconsistent. The noun at the otherside of the underscore shouldn't be lowercased e.g. `CreateIfNoOtherUserInvite_Role`. 

Same for the other test cases

Reply:
I lowercased it to indicate it was referring to the argument variable, to distinguish which overloaded method it's using. I'll still change it.